auto-value-gson-extension/src/main/java/com/ryanharter/auto/value/gson/AutoValueGsonExtension.java

```diff
@@ -869,6 +889,16 @@ private MethodSpec createReadMethod(ClassName className,
       }
     }
 
+    Property unrecognisedJsonPropertiesContainer = properties.stream()
+      .filter(this::isUnrecognisedJsonPropertiesContainer)
+      .findFirst()
```

Comment:
Let's throw an exception if we find more than one field annotated.

Reply:
<img width="1125" alt="image" src="https://user-images.githubusercontent.com/6190346/163579409-1b0d712d-951a-4af3-a6b3-7b8c614392a4.png">

logged this case as an error